packages/richtext-lexical/src/lexical/plugins/TextPlugin/index.tsx

```diff
@@ -0,0 +1,37 @@
+import type { TextFormatType } from 'lexical'
```

Comment:
Can you add 'use client' to the top of this file? React Compiler skips files that do not have that directive

Reply:
done, I don't know why eslint didn't tell me

Reply:
@AlessioGr are there any docs about this?